### PR TITLE
Feature/results serialization

### PR DESCRIPTION
**Pull request by rnugent3**
Based on saving results for a given IAS, I think we have all the serialization we need? I am not sure if we need to go higher than an IContainResults object.

### CODE REVIEW

File: fda-model-test/unittests/ExpectedAnnualDamageResultsShould.cs [hunk 1]
```diff
@@ -66,6 +66,8 @@ public void SerializationShouldReadTheSameObjectItWrites(int seed, int iteration
             double actualMin = expectedAnnualDamageResults.HistogramsOfEADs[category].Min;
             Assert.Equal(expectedMin, actualMin);
 
+            bool success = convergenceCriteria.Equals(expectedAnnualDamageResults)
+
```
Comment (HenryGeorgist) [inline fda-model-test/unittests/ExpectedAnnualDamageResultsShould.cs]: did you forget an assert true?

**Review by HenryGeorgist — APPROVED** [review]
it all generally looks good. Great work. I think we should probably serialize alternative and scenario too right? Or are we saying that is the HEC-FDA UI data storage responsibility?

File: fda-model-test/unittests/ExpectedAnnualDamageResultsShould.cs (hunk 1 above)
Comment (rnugent3) [reply fda-model-test/unittests/ExpectedAnnualDamageResultsShould.cs]: it found its way there eventually

**Comment by rnugent3** [comment]
> it all generally looks good. Great work. I think we should probably serialize alternative and scenario too right? Or are we saying that is the HEC-FDA UI data storage responsibility?

I think we decided we would only store results for a Scenario and would compute Alternative and Alternative Comparisons on the fly. I guess we might need to still store some things? I'll talk to Cody and get back to you.

**Pull request merged**